This patch was developed in-house and has been submitted upstream:
http://bugs.python.org/issue23584

--- Python-2.7.10rc1/Lib/test/test_doctest.py.~1~	2015-05-18 14:53:39.203980905 -0700
+++ Python-2.7.10rc1/Lib/test/test_doctest.py	2015-05-18 14:56:30.278638705 -0700
@@ -2582,7 +2582,7 @@
     >>> fn = tempfile.mktemp()
     >>> with open(fn, 'wb') as f:
     ...     f.write('Test:\r\n\r\n  >>> x = 1 + 1\r\n\r\nDone.\r\n')
-    >>> doctest.testfile(fn, False)
+    >>> doctest.testfile(fn, False, verbose=False)
     TestResults(failed=0, attempted=1)
     >>> os.remove(fn)
 
@@ -2591,7 +2591,7 @@
     >>> fn = tempfile.mktemp()
     >>> with open(fn, 'wb') as f:
     ...     f.write('Test:\n\n  >>> x = 1 + 1\n\nDone.\n')
-    >>> doctest.testfile(fn, False)
+    >>> doctest.testfile(fn, False, verbose=False)
     TestResults(failed=0, attempted=1)
     >>> os.remove(fn)
 
